### DISCUSSION TITLE
[BUG] Set-PnPView is not working for Gallery Type View Formatting using Custom JSON

### DISCUSSION THREAD

**Discussion by rbkharade** [pnp/powershell#2511] [General]
### Reporting an Issue or Missing Feature
I am reporting a bug

### Expected behavior 
Set-PnPView should apply a custom formatting to existing modern SharePoint gallery type view.

### Actual behavior 
I am currently trying to apply a custom Json formatting to existing gallery type view. I first connect to my SharePoint site with Connect-PnPOnline (Tried Interactive, WebLogin , Default & App-only tokens approach). After running Set-PnPView with custom Json, command is getting executed successfully. But not able to get desired formatted view.

### Steps to reproduce behavior
$JSONFormat = @"
{
  "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/tile-formatting.schema.json",
  "formatter": {
    "children": [
      {
        "elmType": "div",
        "style": {
          "border-radius": "2px",
          "box-shadow": "0px 1.6px 3.6px 0 #00000024, 0px 0.3px 0.9px 0 #00000024",
          "font-weight": "800",
          "height": "92%",
          "overflow": "hidden",
          "padding-left": "16px",
          "padding-top": "16px",
          "width": "95%"
        },
        "txtContent": "[$Title]"
      }
    ],
    "elmType": "div"
  },
  "height": "250",
  "width": "350"
}
"@
  
Try
{
    #Connect to site
    Connect-PnPOnline -URL $SiteURL -UseWebLogin
    #Set the View Formatting
    Set-PnPView -List $ListName -Identity $ViewName -Values @{"CustomFormatter" =$JSONFormat.ToString()}
    Write-host "View Format Applied Successfully!" -foregroundcolor Green
}
Catch {
    write-host "Error: $($_.Exception.Message)" -foregroundcolor Red
}

### What is the version of the Cmdlet module you are running?
PnP.PowerShell - 1.11.0     

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows

**Comment by rbkharade** [comment]
Hi @KoenZomers, can you please look into it?

**Comment by gautamdsheth** [comment]
Moving this to a discussion, most likely some issue with your formatting JSON.

Please check these links to properly format it:

https://pnp.github.io/script-samples/spo-list-formatting/README.html?tabs=pnpps

or 

https://beaucameron.com/2020/06/01/json-column-formatting-name-cannot-begin-with-the-character/

**Reply by rbkharade** [reply]
Hi @gautamdsheth , 
Thank you for the reply.

I have checked the JSON for formatting issues . The JSON looks fine. I have applied the same JSON file manually where it is applying the formatting correctly. 
You can test the same by following simple steps - 

1. Creating a simple SharePoint list with default column Title .
2. Create a Gallery view.
3. Select view created in step 2 . Select Format Current View & copy paste above JSON file. It will apply the formatting.
4.  But when tried to do the same thing using PnP PowerShell formatting is not getting applied.

**Comment by JelleC-GMI** [comment]
I have the same issue.
No problem with the List type view only not updating the gallery type view
